README.md

```diff
@@ -0,0 +1,38 @@
+# perceptive-client
+
+Lookup image metadata locally using pHash index
+
+## Summary
+
+This is a [mediachain](https://medium.com/mine-labs/mediachain-483f49cbe37a)
+demo that - given an image - will look up its metadata as stored on IPFS. For
```

Comment:
"The em dash is perhaps the most versatile punctuation mark. Depending on the context, the em dash can take the place of commas, parentheses, or colons—in each case to slightly different effect."